Sources/StreamChatUI/CommonViews/ContainerStackView.swift

```diff
@@ -195,14 +195,14 @@ public class ContainerStackView: UIView {
 
         // Add constraints for distribution
         zip(subviews, subviews.dropFirst()).forEach { lView, rView in
+            guard distribution == .equal else { return }
```

Comment:
we can probably move this `guard` outside the loop ;-)

Reply:
```
if distribution == .equal {
            zip(subviews, subviews.dropFirst()).forEach { lView, rView in
                if axis == .horizontal {
                    customConstraints.append(
                        lView.widthAnchor.constraint(equalTo: rView.widthAnchor)
                    )
                } else {
                    customConstraints.append(
                        lView.heightAnchor.constraint(equalTo: rView.heightAnchor)
                    )
                }
            }
        }
```

Reply:
yeah, I have just done that 👍